int16 sensor_idx
int16 rate
---
int16 rate